sysmonitor@orcus/files/sysmonitor@orcus/3.2/applet.js

```diff
@@ -357,6 +368,21 @@ MyApplet.prototype = {
                     this.graphs[i].setWidth(this.getGraphWidth(i), this.vertical);
     },
 
+    on_cfg_changed_tooltip_decimals: function(decimals, graph_idx) {
+        if (graph_idx) {
+            let provider = this.graphs[graph_idx].provider;
+            if ("setTextDecimals" in provider)
+                provider.setTextDecimals(this.getGraphTooltipDecimals(graph_idx));
+        }
+        else
+            for (let i = 0; i < this.graphs.length; i++)
+                if (this.graphs[i]) {
+                    let provider = this.graphs[graph_idx].provider;
```

Comment:
Here, the index defined by the loop (`i`) should be used, not the `graph_idx` argument. The same [on line 382](https://github.com/linuxmint/cinnamon-spices-applets/pull/1864/files#diff-993e16c91327e84ab0a63ce52e970e85R382).

:+1: